submissions/DmytryjK/HTML-CSS-Popup/sass/style.scss

```diff
@@ -0,0 +1,19 @@
+@import 'variables/variables.scss';
+@import 'components/fonts.scss';
+
+* {
```

Comment:
Move these CSS rules into an independent module `base.scss` and leave here just imports. 
Hard to read code when CSS rules are mixed with Sass imports.